com.github.rajsolai.textsnatcher.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "40"
```

Comment:
```suggestion
runtime-version: "41"
```

Reply:
Sure I will update the runtime version :+1: